### PR TITLE
AlexThüring.DVDStyler version 3.1

### PR DESCRIPTION
**Pull request by tiziodcaio**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16237)

### CODE REVIEW

File: manifests/a/AlexThüring/DVDStyler/3.1/AlexThüring.DVDStyler.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Created using wingetcreate
+
+PackageIdentifier: AlexThüring.DVDStyler
```
Comment (mddtest) [inline manifests/a/AlexThüring/DVDStyler/3.1/AlexThüring.DVDStyler.installer.yaml:3]:
```suggestion
PackageIdentifier: AlexThUring.DVDStyler

```

File: manifests/a/AlexThüring/DVDStyler/3.1/AlexThüring.DVDStyler.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate
+
+PackageIdentifier: AlexThüring.DVDStyler
```
Comment (mddtest) [inline manifests/a/AlexThüring/DVDStyler/3.1/AlexThüring.DVDStyler.locale.en-US.yaml:3]:
```suggestion
PackageIdentifier: AlexThUring.DVDStyler

```

File: manifests/a/AlexThüring/DVDStyler/3.1/AlexThüring.DVDStyler.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate
+
+PackageIdentifier: AlexThüring.DVDStyler
+PackageVersion: 3.1
+PackageLocale: en-US
+Publisher: Alex Thüring
```
Comment (mddtest) [inline manifests/a/AlexThüring/DVDStyler/3.1/AlexThüring.DVDStyler.locale.en-US.yaml:6]: PackageIdentifier: AlexThUring.DVDStyler

File: manifests/a/AlexThüring/DVDStyler/3.1/AlexThüring.DVDStyler.yaml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: AlexThüring.DVDStyler
```
Comment (mddtest) [inline manifests/a/AlexThüring/DVDStyler/3.1/AlexThüring.DVDStyler.yaml:3]:
```suggestion
PackageIdentifier: AlexThUring.DVDStyler

```

**Comment by mddtest** [comment]
using AlexThüring.DVDStyler may cause some issues when searching using AlexThUring.DVDStyler would be better

**Comment by tiziodcaio** [comment]
> 
> 
> using AlexThüring.DVDStyler may cause some issues when searching using AlexThUring.DVDStyler would be better

I will do, but I have to do a commit cuz I have to change the folder too

File: manifests/a/AlexThuering/DVDStyler/3.1/AlexThuering.DVDStyler.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,34 @@
+# Created using wingetcreate
+
+PackageIdentifier: AlexThuering.DVDStyler
```
Comment (ItzLevvie) [inline manifests/a/AlexThuering/DVDStyler/3.1/AlexThuering.DVDStyler.installer.yaml:3]:
```suggestion
PackageIdentifier: AlexThuring.DVDStyler
```

File: manifests/a/AlexThuering/DVDStyler/3.1/AlexThuering.DVDStyler.installer.yaml [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+- silent
+- silentWithProgress
+FileExtensions:
+- dvds xml mp4 mp3 wav mkv
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: exe
+  Scope: machine
+  InstallerUrl: https://downloads.sourceforge.net/project/dvdstyler/dvdstyler/3.1/DVDStyler-3.1-win64.exe
```
Comment (ItzLevvie) [inline manifests/a/AlexThuering/DVDStyler/3.1/AlexThuering.DVDStyler.installer.yaml:17]:
```suggestion
  InstallerUrl: https://sourceforge.net/projects/dvdstyler/files/dvdstyler/3.1/DVDStyler-3.1-win64.exe/download
```

File: manifests/a/AlexThuering/DVDStyler/3.1/AlexThuering.DVDStyler.installer.yaml [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+  InstallerSha256: 85D91E6C8B17D56E9EC1CAFB2723C889E0220F19EDCE41FCC9B992DB83B8265F
+  InstallerSwitches:
+    Silent: /verysilent
+    SilentWithProgress: /silent
+  UpgradeBehavior: install
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: exe
+  Scope: machine
+  InstallerUrl: https://downloads.sourceforge.net/project/dvdstyler/dvdstyler/3.1/DVDStyler-3.1-win32.exe
```
Comment (ItzLevvie) [inline manifests/a/AlexThuering/DVDStyler/3.1/AlexThuering.DVDStyler.installer.yaml:27]:
```suggestion
  InstallerUrl: https://sourceforge.net/projects/dvdstyler/files/dvdstyler/3.1/DVDStyler-3.1-win32.exe/download
```

File: manifests/a/AlexThuering/DVDStyler/3.1/AlexThuering.DVDStyler.locale.en-US.yaml [hunk 8]
```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate
+
+PackageIdentifier: AlexThuering.DVDStyler
```
Comment (ItzLevvie) [inline manifests/a/AlexThuering/DVDStyler/3.1/AlexThuering.DVDStyler.locale.en-US.yaml:3]:
```suggestion
PackageIdentifier: AlexThuring.DVDStyler
```

File: manifests/a/AlexThuering/DVDStyler/3.1/AlexThuering.DVDStyler.locale.en-US.yaml [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+PackageVersion: 3.1
+PackageLocale: en-US
+Publisher: Alex Thuering
```
Comment (ItzLevvie) [inline manifests/a/AlexThuering/DVDStyler/3.1/AlexThuering.DVDStyler.locale.en-US.yaml:6]:
```suggestion
Publisher: Thüring IT-Consulting
```

File: manifests/a/AlexThuering/DVDStyler/3.1/AlexThuering.DVDStyler.yaml [hunk 10]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: AlexThuering.DVDStyler
```
Comment (ItzLevvie) [inline manifests/a/AlexThuering/DVDStyler/3.1/AlexThuering.DVDStyler.yaml:3]:
```suggestion
PackageIdentifier: AlexThuring.DVDStyler
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Other than the changes I've made, everything else looks good to me:
image

File: manifests/a/AlexThuering/DVDStyler/3.1/AlexThuering.DVDStyler.yaml (hunk 10 above)
Comment (tiziodcaio) [reply manifests/a/AlexThuering/DVDStyler/3.1/AlexThuering.DVDStyler.yaml:3]: This is the german pronunciation

Comment (tiziodcaio) [reply manifests/a/AlexThuering/DVDStyler/3.1/AlexThuering.DVDStyler.yaml:3]: ü --> ue

Comment (ItzLevvie) [reply manifests/a/AlexThuering/DVDStyler/3.1/AlexThuering.DVDStyler.yaml:3]: Okay, that is fine :)

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**